frontend/src/components/Reports/Routine.js

```diff
@@ -129,7 +129,7 @@ export const RoutineReportsMenu = {
       icon: IbmWatsonNaturalLanguageUnderstanding,
       SideNavMenuItem: [
         {
-          link: config.serverBaseUrl + "/AuditTrailReport",
+          link: "/RoutineReport/AuditTrailReport",
```

Comment:
Here it should be `link: "/AuditTrailReport",` just same as you have done for study.js below

Reply:
Hi @manishjha-04 
I did that but was still getting redirected to `ttps://localhost/api/OpenELIS-Global/AuditTrailReport`

Reply:
Have you created route in app.js for  /AuditTrailReport ,that i asked you for?

Reply:
Yes
I just did it now and it works
Thanks a bunch